compiler/rustc_borrowck/src/diagnostics/explain_borrow.rs

```diff
@@ -102,14 +99,14 @@ impl<'tcx> BorrowExplanation<'tcx> {
             BorrowExplanation::UsedLaterInLoop(later_use_kind, var_or_use_span, path_span) => {
                 let message = match later_use_kind {
                     LaterUseKind::TraitCapture => {
-                        "borrow captured here by trait object, in later iteration of loop"
+                        "borrow is captured here by trait object, in later iteration of loop"
```

Comment:
Adding a verb is inconsistent with the current phrasing of the messages in the previous branch.  I'd rather have this change in a specific PR, to ensure consistency there.

Reply:
Sure.